zebra-chain/src/transaction/arbitrary.rs

```diff
@@ -778,7 +778,9 @@ impl Arbitrary for Transaction {
             NetworkUpgrade::Blossom | NetworkUpgrade::Heartwood | NetworkUpgrade::Canopy => {
                 Self::v4_strategy(ledger_state)
             }
-            NetworkUpgrade::Nu5 => prop_oneof![
+            // TODO: move `NetworkUpgrade::Nu6` to its own match arm if there's a new tx format for
+            // NU6 or remove this TODO if there isn't.
```

Comment:
Optional/Nitpick: It's almost certainly too late to update the NU6 transaction format at this point if we want to remove this TODO now

```suggestion
```